src/test/regress/expected/query_single_shard_table.out

```diff
@@ -1398,6 +1398,7 @@ INSERT INTO nullkey_c1_t1 SELECT * FROM nullkey_c1_t2;
 SET client_min_messages TO DEBUG2;
 --    between two non-colocated single-shard tables
 INSERT INTO nullkey_c1_t1 SELECT * FROM nullkey_c2_t1;
+DEBUG:  Creating router plan
```

Comment:
@onurctirtir can you please review the test output changes and let me know if there are any unexpected changes.

I already did a pass, but would be nice to get a review from you.

Also please check insert_select_single_shard.out changes as well